pages/develop.md

```diff
@@ -205,16 +205,26 @@ Adding a development extension to Firefox is more complex than adding one to
 Chrome. You must add the extension to a Firefox "profile."
 
 1. Clone the Git Repository, `git clone --recursive https://github.com/privly/privly-firefox.git`
-1. In your terminal, find your Firefox profile directory.  
+2. In your terminal, find your Firefox profile directory.  
 [Windows](http://support.mozilla.org/en-US/questions/941548),  
 Mac: `cd ~/Library/Application\ Support/Firefox/profiles/`  
 Linux:  `cd ~/.mozilla/`  
 In that location, you will find your Firefox profile directories, e.g. 12a3bc4d.dev., change directories to your profile folder, for example `cd 12a3bc4d.dev`
-1. Create a file to include a pointer to your code. For this, you need the path of where you cloned the Git Repository for the extension. Use vim (or your preferred editor) to create a filename with privly@priv.ly. On some systems (no, we're not sure which ones) this won't work and your filename will need to be {ec8030f7-c20a-464f-9b0e-13a3a9e97384}.
-1. Type in the path your newly cloned git repository from earlier, e.g. C:\path\to\extension\privly-firefox\ (Windows) or /path/to/extension/privly-firefox/ (Linux, Mac), and save the file. 
-1. Open Firefox.
-1. You should see a dialog asking to confirm the Privly extension installation. 
-1. Success! If it didn't work, see the tip from above, where the filename in your development profile folder may need to be privly@priv.ly or {ec8030f7-c20a-464f-9b0e-13a3a9e97384}.
+3. Create a file to include a pointer to your code. For this, you need the path of where you cloned the Git Repository for the extension. Use vim (or your preferred editor) to create a filename with privly@priv.ly. On some systems (no, we're not sure which ones) this won't work and your filename will need to be {ec8030f7-c20a-464f-9b0e-13a3a9e97384}.
```

Comment:
It is best not to change these numbers. They will be numbered sequentially when rendered from Markdown. See: https://priv.ly/pages/develop.html#FirefoxExtension